nx/lib/nx.ex

```diff
@@ -6788,6 +6788,53 @@ defmodule Nx do
         [2.0, 6.0, 10.0]
       >
 
+      iex> t = Nx.tensor([
+      ...>             [
+      ...>               0.7731901407241821,
+      ...>               0.5813425779342651,
+      ...>               0.8365984559059143,
+      ...>               0.2182593196630478,
+      ...>               0.06448899209499359,
+      ...>               0.9420031905174255
+      ...>             ],
+      ...>             [
+      ...>               0.6547101736068726,
+      ...>               0.05023770406842232,
+      ...>               0.657528281211853,
+      ...>               0.24924135208129883,
+      ...>               0.8238568902015686,
+      ...>               0.11182288080453873
+      ...>             ],
+      ...>             [
+      ...>               0.7693489193916321,
+      ...>               0.6696648001670837,
+      ...>               0.6877049803733826,
+      ...>               0.08740159869194031,
+      ...>               0.6053816676139832,
+      ...>               0.5419610142707825
+      ...>             ],
+      ...>             [
+      ...>               0.03419172018766403,
+      ...>               0.8298202753067017,
+      ...>               0.6097439527511597,
+      ...>               0.0184243805706501,
+      ...>               0.5578944087028503,
+      ...>               0.9986271858215332
+      ...>             ]
+      ...>           ]
+      ...> )
+      iex> weights = Nx.tensor([
+      ...>  0.8669093251228333,
+      ...>  0.10421276837587357,
+      ...>  0.996828556060791,
+      ...>  0.29747673869132996
+      ...> ])
```

Comment:
This example is not very useful for someone reading the documentation. I think we should:

1. Convert this example to integers, so it is more readable
2. Move it to nx_test.exs

WDYT?

Reply:
The only important thing in the example is shape of tensors, so I'll choose the first option

Reply:
The current one was the example which caused an error in linear regression testing and I paste it straightforwardly

Reply:
I see. Let's go ahead and do both changes (i.e. change the example and move it to a test file!).